pkg/gatherers/clusterconfig/gather_cluster_version_test.go

```diff
@@ -0,0 +1,134 @@
+package clusterconfig
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+
+	"github.com/openshift/insights-operator/pkg/utils/marshal"
+
+	configFake "github.com/openshift/client-go/config/clientset/versioned/fake"
+	"k8s.io/client-go/kubernetes/fake"
+
+	configv1 "github.com/openshift/api/config/v1"
+
+	"github.com/openshift/insights-operator/pkg/record"
+	"github.com/stretchr/testify/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func Test_getClusterVersion(t *testing.T) {
+	lastTimestampEvent := metav1.Time{Time: time.Now().Add(2)}
+	compactEvents := eventListToCompactedEventList(&v1.EventList{
```

Comment:
Basically, it needs to be here, 'cause when it passes to the `JSONMarshaller` if I don't pass it as a pointer, it doesn't match with the way that the info is generated by the gatherer.